dotnet/sidebars.js

```diff
@@ -10,8 +10,10 @@ module.exports = {
         { "type": "doc", "id": "cli" },
         { "type": "doc", "id": "debug" },
         { "type": "doc", "id": "languages" },
+        { "type": "doc", "id": "showcase" },
```

Comment:
Our show case was very JS-centric. Also I don't think we want show case in our docs at all.